app/helpers/application_helper.rb

```diff
@@ -15,15 +15,16 @@ def multiple_with_filter?
   end
 
   # this helper should be used to print date time in absolute form
+  # unless you pass show_relative_time_tooltip = false
```

Comment:
I am not a native speaker, but for a while it made me think that what is behind 'unless' relates to the previous line.

What do you think about merging it with the following line:

```js
// it will also define a title with relative time information unless you pass show_relative_time_tooltip = false
```

or even below it:

```js
// it will also define a title with relative time information
// unless you pass show_relative_time_tooltip = false
```

Reply:
@jeremylenz What do you think is the correct way here?

Reply:
The grammar is ambiguous here since no punctuation was used.  It could mean
```
// This helper should be used to print date/time in absolute form,
// unless you pass show_relative_time_tooltip = false.
// It will also define a title with relative time information.
```
or
```
// This helper should be used to print date/time in absolute form.
// Unless you pass show_relative_time_tooltip = false,
// it will also define a title with relative time information.
```
Looking at the code, I think @xprazak2 is right and the second one is correct.

To avoid any ambiguity at all, you could both change the order as per @xprazak2's suggestion _and_ add correct punctuation: 
```
// This helper should be used to print date/time in absolute form.
// It will also define a title with relative time information,
// unless you pass show_relative_time_tooltip = false.
```

Reply:
Thanks! changed to the last option